### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RubenFalcao14**
I have added the required full stops at various places in the read.me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Doc2Pen
+# Doc2Pen.
```
Comment (smaranjitghose) [inline README.md:1]: This is a heading😅

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 An open source project aimed at making your student life easier!
 
 
-## DEMO
+## DEMO.
```
Comment (smaranjitghose) [inline README.md:6]: This is a sub-heading😅

File: README.md [hunk 3]
```diff
@@ -13,61 +13,61 @@ Add GIFs of the entire web app
...
-- [React Reveal](https://www.react-reveal.com/): Animations
-- [JSPDF](https://www.npmjs.com/package/jspdf): Generate PDFs
-- [React Scroll](https://www.npmjs.com/package/react-scroll): Smooth Scrolling between sections
+- React.
+- HTML.
+- SCSS.
+- [React - Bootstrap](https://react-bootstrap.github.io/).
+- [React - Helmet](https://www.npmjs.com/package/react-helmet): Document Head Management.
+- [React - Leaflet.js](https://react-leaflet.js.org/): Interactive Maps.
+- [React - Dropzone.js](https://react-dropzone.js.org/): Drag and drop file uploads.
```
Comment (smaranjitghose) [inline README.md:64]: This is a list + Comments

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
PR not relevant

**Comment by RubenFalcao14** [comment]
Sorry 😓

**Pull request closed without merging**